shlr/meson.build

```diff
@@ -233,6 +233,92 @@ sdb_gen_cmd = [
 ]
 
 
+# handle tree-sitter dependency
+tree_sitter_path = join_paths(meson.current_source_dir(), 'tree-sitter')
```

Comment:
Should be under `meson.current_build_dir()`. Meson should never touch the source dir. If it has to be in the source dir, submodules should be used instead.

Reply:
i know, you are right, but... i was following what was requested to be done with capstone as well. I'm perfectly fine with moving it to current_build_dir, but I'm pretty sure when I will implement this in acr/makefile it will live in the source directory (as there is no "build" directory in that case).

Reply:
Yeah acr being in-source is bad by itself. But if it works, at least meson should stay in the build dir.

Reply:
yes, it can easily work for meson! I'll do it.